docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1a.md

```diff
@@ -83,15 +83,15 @@ Also: if you ever need to look up just one mailbox workspace by
 SyncTarget name, you could do it as follows.
 
 ```shell
-GUILDER_WS=$(kubectl get Workspace -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "guilder") | .name')
+GUILDER_WS=$(kubectl get Workspace -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] | tostring | endswith("guilder")) | .name')
```

Comment:
This line should not have to change.
The annotation value is for users, so should be the user's name for the SyncTarget (i.e., the name in the IS).

Reply:
Changed back. I remember that I saw the names in ESPW at the very beginning of this PR. But now somehow I saw names in IS now.

Reply:
done